dimod/response.py

```diff
@@ -74,14 +74,15 @@ class Response(Iterable, Sized):
         {'a': -1, 'b': 1} 1.0
 
     """
+    _REQUIRED_FIELDS = ['sample', 'energy', 'num_occurrences']
```

Comment:
Even better, make it a `set` here, you use it for membership testing later on, either way.

Reply:
Need it to be ordered